config/utils.go

```diff
@@ -44,10 +44,14 @@ func init() {
 
 	path := "cache"
 
-	if _, err := os.Stat(ProcessPath + PATH_SEPARATOR + path); errors.Is(err, os.ErrNotExist) {
-		err := os.Mkdir(ProcessPath+PATH_SEPARATOR+path, os.ModePerm)
-		if err != nil {
-			log.Println(err)
+	var imageTypes []string = []string{"grids", "heroes", "logos", "icons"}
```

Comment:
can this array made public so it can be used in `controller/grid.go` line 29 as replacement of the many conditions for the if statement?

Reply:
Exported a function that checks if provided value is one of those. That should do it I guess

Reply:
> Exported a function that checks if provided value is one of those. That should do it I guess

perfect! so we have only one list to keep updated